packages/demand-bloc-track-trip/src/tripTransformer.ts

```diff
@@ -108,7 +109,8 @@ export const tripTransformer = (trip: OriginalTripFollowResponse | BookATripResp
     termsAndConditionsUrl: dv(fleet_info?.terms_conditions_url),
     supplierLogoUrl: dv(fleet_info?.logo_url),
     fleetId: dv(fleet_info?.fleet_id),
-    vehicleClass: dv(quote?.vehicle_class),
+    vehicleClass: dv(vehicle?.vehicle_class || quote?.vehicle_class),
+    vehicleType: dv(vehicle?.type),
```

Comment:
Seems fleet is not best place to put vehicle details.
I see currently vehicle props are spread across multiple sections `fleet` and `driver`

I suggest to create new `vehicle` section which will contain all vehicle related props.
What do you think?
We can keep all existing sections untouched (some vehicle props. will be duplicated)

Reply:
Let's also expose vehicle tags